pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[tool.black]
+line-length = 79
+# NOTE: PEP8 does recommend double-quotes, but to start with we'll support
+# single quotes as to minimize the patch/noise produced by introducing black
+skip-string-normalization = true
```

Comment:
I decided to go for 79 chars, as it's by the PEP8 standard.
Also, I figured - for now - to allow using single quote strings. We can change it later if we want to.